src/video_call.c

```diff
@@ -258,6 +266,47 @@ void callback_video_end(uint32_t friend_number)
 /*
  * Commands from chat_commands.h
  */
+void cmd_vcall(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[MAX_STR_SIZE])
+{
+    UNUSED_VAR(window);
+    UNUSED_VAR(m);
+    UNUSED_VAR(argv);
+
+    const char *error_str;
+
+    if (argc != 0) {
+        error_str = "Unknown arguments.";
+        goto on_error;
+    }
+
+    if (!CallControl.av) {
+        error_str = "ToxAV not supported!";
+        goto on_error;
+    }
+
+    if (!self->stb->connection) {
+        error_str = "Friend is offline.";
+        goto on_error;
+    }
+
+    Call *call = &CallControl.calls[self->num];
+
+    if (call->status != cs_None) {
+        error_str = "Already calling.";
+        goto on_error;
+    }
+
+    init_call(call);
+
+    call->video_bit_rate = DEFAULT_VIDEO_BIT_RATE;
+
+    place_call(self);
+
+    return;
+on_error:
```

Comment:
This goto doesn't seem worth it to me. There isn't any cleanup we're doing in the on_error part, and
```
print_err(self, "oh no something bad"); return;
```
is shorter than
```
error_str = "oh no something bad"; goto on_error;
```
while the latter costs more time to understand.